src/java/picard/metrics/GcBiasMetrics.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2009 The Broad Institute
```

Comment:
:) the original code is very old, I didn't change the calculations it does, just so that it works with single pass sam and the multi level collector. Should I just change this to 2015 or is there a new copyright statement I should paste in?